plugin/trino-pinot/src/main/java/io/trino/plugin/pinot/PinotColumnHandle.java

```diff
@@ -128,9 +129,12 @@ public static Type getTrinoTypeFromPinotType(FieldSpec.DataType dataType)
             case LONG:
                 return BigintType.BIGINT;
             case STRING:
+            case JSON:
```

Comment:
Should this be JsonType or VarcharType? How JSON functions will be handled here e.g. json_extract ?

Reply:
I was going to leave functions out of the scope of this PR but I'll check how easy it would be to include them otherwise I'll raise a separate issue for this. Probably the only 2 function that could be easily mapped are `json_extract_scalar` and `json_extract_scalar` as each has an equivalent in Pinot. I'll get back to you on this

Reply:
I see, can you test it locally with [PinotJsonQuickStart](https://github.com/apache/pinot/blob/master/pinot-tools/src/main/java/org/apache/pinot/tools/JsonIndexQuickStart.java)?
Here is a reference PR as well.
https://github.com/prestodb/presto/pull/17015/files#diff-379277f1690c545f2f182815aa2b984399e26318df7e2402be7149a2c5e035b4R135

https://github.com/prestodb/presto/pull/17323

Reply:
Thanks @xiangfu0 I'll take a look at those. In the meantime it seems I've found an issue with multi-value timestamp columns, I've raised an issue in the pinot project https://github.com/apache/pinot/issues/8624

Reply:
You can get `JsonType` via `TypeManager`. `PostgreSqlClient` has the example.

Reply:
I think it would be better to revert this change and implement in #13428